packages/node/test/handlers.test.ts

```diff
@@ -281,6 +283,31 @@ describe('requestHandler', () => {
       done();
     });
   });
+
+  it('patches `res.end` when `flushTimeout` is specified', () => {
+    const flush = jest.spyOn(SDK, 'flush').mockResolvedValue(true);
+
+    const handler = requestHandler({ flushTimeout: 1337 });
+    handler(req, res, next);
+    res.end('ok');
+
+    setImmediate(() => {
+      expect(flush).toBeCalledWith(1337);
+      expect(res.finished).toBe(true);
+    });
+  });
+
+  it('prevents errors thrown during `flush` from breaking the response', async () => {
+    jest.spyOn(SDK, 'flush').mockRejectedValue(new SentryError('HTTP Error (429)'));
+
+    const handler = requestHandler({ flushTimeout: 1337 });
+    handler(req, res, next);
+    res.end('ok');
+
+    setImmediate(() => {
+      expect(res.finished).toBe(true);
+    });
+  });
```

Comment:
Added a non-flush-timeout test and made the language more consistent with other tests in the file.

```suggestion
  it('patches `res.end`', () => {
    const flush = jest.spyOn(SDK, 'flush').mockResolvedValue(true);

    sentryRequestMiddleware(req, res, next);
    res.end('ok');

    setImmediate(() => {
      expect(flush).toHaveBeenCalled();
      expect(res.finished).toBe(true);
    });
  });
  
  it('patches `res.end` when `flushTimeout` is specified', () => {
    const flush = jest.spyOn(SDK, 'flush').mockResolvedValue(true);

    const sentryRequestMiddleware = requestHandler({ flushTimeout: 1337 });
    sentryRequestMiddleware(req, res, next);
    res.end('ok');

    setImmediate(() => {
      expect(flush).toHaveBeenCalledWith(1337);
      expect(res.finished).toBe(true);
    });
  });

  it('prevents errors thrown during `flush` from breaking the response', async () => {
    jest.spyOn(SDK, 'flush').mockRejectedValue(new SentryError('HTTP Error (429)'));

    sentryRequestMiddleware(req, res, next);
    res.end('ok');

    setImmediate(() => {
      expect(res.finished).toBe(true);
    });
  });
```